README.md

```diff
@@ -121,6 +121,9 @@ struct ContentView: View {
            .onPixelEvent { event in
              handlePixelEvent(event)
            }
+           .onTouchPayButton {
```

Comment:
I think the name of the handler should be `onPayButtonTouch` (not important, just to be consistent with the naming of the other handlers like `onLinkClick`).